packages/toolboxes/cosmos/src/toolbox/kujira.ts

```diff
@@ -42,5 +46,32 @@ export const KujiraToolbox = ({ server }: ToolboxParams = {}): KujiraToolboxType
         fastest: new SwapKitNumber({ value: baseFee * 2, decimal: BaseDecimal.KUJI }),
       };
     },
+    getBalance: async (address: string, _potentialScamFilter?: boolean) => {
+      const denomBalances = await client.getBalance(address);
+      return await Promise.all(
+        denomBalances
+          .filter(({ denom }) => {
+            if (!denom) {
+              return false;
+            }
+            if (denom.includes('IBC/')) {
+              return false;
+            }
+            // include USK from Kujira
+            if (
+              denom ===
+              'FACTORY/KUJIRA1QK00H5ATUTPSV900X202PXX42NPJR9THG58DNQPA72F2P7M2LUASE444A7/UUSK'
+            ) {
+              return true;
+            }
+            // exclude all other FACTORY tokens
+            if (denom.startsWith('FACTORY')) {
+              return false;
+            }
+            return true;
+          })
```

Comment:
That could be returned as boolean value. Will refactor before release.

Reply:
done

Reply:
@phips28 I thought of all statements but thanks ^^

Reply:
nooo 🙈 this gets out of hands and is un-readable/maintainable when this list grows.

let the compiler optimize it 😉

Reply:
@phips28 sorry to disagree 😅 - We will keep code the way we are doing rest in our codebase. It isn't clearer that way with random strings in code.

Reply:
Its not a random string, its a contract. 😉

Reply:
Yeah - I was going to use it as constant in `cosmos` package - more descriptive to have it as `USK_FACTORY_CONTRACT` or sth like that 😅.
Thanks for contribution 🙏 - it should be released in next rc version in like an hour or so :)

Reply:
Very good idea 👍 
Thank you!